.gitlab/benchmarks/microbenchmarks.yml

```diff
@@ -138,3 +139,60 @@ run-benchmarks:
     - ./platform/steps/analyze-results.sh
     - ./platform/steps/post-pr-comment.sh
     - ./platform/steps/upload-to-bp-ui.sh
+
+# This repository is using PR-level performance quality gates.
+# Verify that the check-big-regressions CI job has passed. If any regression happened, merging this PR will be blocked.
+# If bypassing is necessary, see https://datadoghq.atlassian.net/wiki/x/8YFzMwE for more details.
+.check-big-regressions:
+  stage: gate
+  tags: ["arch:amd64"]
+  image: registry.ddbuild.io/images/benchmarking-platform-tools-ubuntu:latest
+  needs: ["run-benchmarks"]
+  # We set when: always when applicable to ensure the job runs even if the run-benchmarks job fails.
+  rules:
+    - if: $CI_COMMIT_REF_NAME =~ /^v[0-9]+\.[0-9]+\.[0-9]+(-prerelease)?$/
+      when: never
+    - if: $CI_COMMIT_REF_NAME == "master"
+      interruptible: false
+      when: always
+    - interruptible: true
+      when: always
+  artifacts:
+    name: "artifacts"
+    when: always
+    paths:
+      - reports/
+    expire_in: "30 days"
+  variables:
+    BP_INFRA_BENCHMARKING_PLATFORM_BRANCH: "dd-trace-dotnet/micro"
+
+    # Regex to ignore benchmarks identified as unstable
+    IGNORED_BENCHMARKS_REGEX: "Trace.Iast.StringAspectsBenchmark|Trace.CharSliceBenchmark|Trace.Asm.AppSecWafBenchmark|Trace.Asm.AppSecBodyBenchmark|Trace.CIVisibilityProtocolWriterBenchmark"
```

Comment:
Checking hte jobs that are more flaky, we seem that we are mostly covered. Still some of the netcoreapp3.1 ones are not included in this list. I would say that, for now, we should exclude any netcoreapp3.1 test. Is that possible?